platform/autoupdate.services/nbproject/org-netbeans-modules-autoupdate-services.sig

```diff
@@ -157,6 +158,7 @@ CLSS public final static !enum org.netbeans.api.autoupdate.OperationException$ER
 fld public final static org.netbeans.api.autoupdate.OperationException$ERROR_TYPE ENABLE
 fld public final static org.netbeans.api.autoupdate.OperationException$ERROR_TYPE INSTALL
 fld public final static org.netbeans.api.autoupdate.OperationException$ERROR_TYPE INSTALLER
+fld public final static org.netbeans.api.autoupdate.OperationException$ERROR_TYPE MISSING_UNPACK200
```

Comment:
Added as part of #2317 and btw. the support works: this week I managed to install old [NBPython](http://plugins.netbeans.org/plugin/61688/python) support into NetBeans 12.2beta2 running on JDK14.